pkg/grpc/session/session.go

```diff
@@ -28,3 +39,17 @@ func Get(ctx context.Context, client databroker.DataBrokerServiceClient, session
 	}
 	return &s, nil
 }
+
+// Set sets a session in the databroker.
+func Set(ctx context.Context, client databroker.DataBrokerServiceClient, s *Session) (*databroker.SetResponse, error) {
+	any, _ := anypb.New(s)
+	res, err := client.Set(ctx, &databroker.SetRequest{
+		Type: any.GetTypeUrl(),
+		Id:   s.Id,
+		Data: any,
+	})
+	if err != nil {
+		return nil, err
```

Comment:
Should we wrap the error to provide additional context? Otherwise, we can simply return client.Set() I think